### PR TITLE
[16.0][ADD] partner_country_state_required

### PR DESCRIPTION
**Pull request by OriolMForgeFlow**

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Technical review LGTM

Readme seems to be from a different module, but it will be overridden anyway

File: partner_country_state_required/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module forces partners assigned to countries having fed. states should have a state assigned.
```
Comment (JordiMForgeFlow) [inline partner_country_state_required/readme/DESCRIPTION.rst]:
```suggestion
This module makes the country state required on partners assigned to countries having fed. states.
```

Comment (OriolMForgeFlow) [reply partner_country_state_required/readme/DESCRIPTION.rst]: Applied!

**Review by JordiMForgeFlow — APPROVED** [review]
Code + functional review LGTM 👍🏼

**Review by SergiCForgeFlow — APPROVED** [review]
Code review LGTM :+1:

**Comment by JordiMForgeFlow** [comment]
I have rebased the PR to resolve conflicts CC @OriolMForgeFlow

**Review by JordiMForgeFlow — APPROVED** [review]
Thanks for the new improvements @OriolMForgeFlow , code + functional review LGTM 👍🏼

**Pull request merged**